### ISSUE TITLE
Upgrade to ui-router v1.x

### ISSUE THREAD

**Issue by Toub** [FountainJS/generator-fountain-webapp#65]
## Description

Current version of angular ui-router (0.3.x) does not support to route to angular 1.5 components. 

However, last version (currently 1.0.0-beta.1) supports a new 'component' attribute, with various syntaxes. 

Let's upgrade, or use this polyfill: https://github.com/softsimon/ui-router-route-to-components

We should also add an example in the gererator.

e.g. for a component named "fleet":

fleet/fleet.js

``` js
module.exports = {
  templateUrl: 'app/fleet/fleet.html',
  controller: Fleet,
  bindings: {
    todos: '=',
    filter: '<'
  }
};

/** @ngInject */
function Fleet($log) {
  this.$onInit = function () {
    $log.info('Hi there!');
  };
}
```

index.js

``` js
var FleetComponent = require('./app/fleet/fleet');
// [...]
angular
  .component('fleetComponent', FleetComponent);
```

route.js

``` js
.state('fleet', {
            url: '/fleet',
            component: 'fleetComponent'
        })
```
### Config

``` json
{
  "generator-fountain-angular1": {
    "version": "0.5.4",
    "props": {
      "framework": "angular1",
      "modules": "webpack",
      "js": "js",
      "css": "scss",
      "resolved": "/usr/local/lib/node_modules/generator-fountain-webapp/node_modules/generator-fountain-angular1/generators/app/index.js",
      "namespace": "fountain-angular1",
      "argv": {
        "remain": [],
        "cooked": [],
        "original": []
      },
      "skip-cache": false,
      "skip-install": false,
      "sample": "todoMVC",
      "router": "uirouter"
    }
  }
}
```
## Relevant Links
- https://github.com/angular-ui/ui-router/issues/2627
- https://github.com/angular-ui/ui-router/issues/2547

**Comment by zckrs** [comment]
releated line code 

https://github.com/FountainJS/generator-fountain-angular1/blob/master/generators/app/index.js#L59

**Comment by micaelmbagira** [comment]
https://github.com/FountainJS/generator-fountain-webapp/issues/65

**Comment by madasuk** [comment]
Hi, can any one update when will be the next release so that i can assume that this feature is available?

**Comment by micaelmbagira** [comment]
Fixed in 0.7.0 (just released) 😉  @madasuk
